### ISSUE TITLE
Accessibility concern for onChange events on select lists?

### ISSUE THREAD

**Issue by num3thod** [wet-boew/GCWeb#980]
@LaurentGoderre @thomasgohard 
Any major show stoppers if I was to implement an onChange event (redirection) on a select list?

Example here: http://jsfiddle.net/num3thod/bbars2nj/1/ (scroll down to the select list example)

Thanks.

**Comment by LaurentGoderre** [comment]
If it's a redirection, then it's a failure of WCAG SC 3.3.2 On Input

http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html

**Comment by num3thod** [comment]
Merci.
